ansible/kp_upload-schema.yml

```diff
@@ -17,8 +17,19 @@
       vars:
         blob_container_name: "{{ plugin_storage }}"
         container_public_access: "container"
-        blob_container_folder_path: "/schemas/local"
+        blob_container_folder_path: "schemas/local"
```

Comment:
Can you please let me know why we are introducing this change?

Reply:
@keshavprasadms  that was a mistake, the change is suppose to be related to  GCP only as it treats  `/` as a  new folder  , reverted it , pls check